bin/logstash.lib.sh

```diff
@@ -1,4 +1,9 @@
-basedir=$(cd `dirname $0`/..; pwd)
+unset CDPATH
```

Comment:
You remove it in ```bin/logstash```, if gone from there, why not from here? or the other way around.